client/src/featureform/client.py

```
@@ -29,6 +30,7 @@
     SourceRegistrar,
     SubscriptableTransformation,
     TrainingSetVariant,
+    ColumnResource,
```

Ruff (0.8.2)

`.register.ColumnResource` imported but unused

Remove unused import: `.register.ColumnResource`

(F401)